angular.json

```diff
@@ -32,9 +32,17 @@
               }
             ],
             "styles": [
-              "apps/codelab/src/styles.scss"
+              "apps/codelab/src/styles.scss",
+              "node_modules/prismjs/themes/prism-okaidia.css",
+              "node_modules/prismjs/plugins/line-numbers/prism-line-numbers.css"
             ],
-            "scripts": []
+            "scripts": [
+              "node_modules/marked/lib/marked.js",
+              "node_modules/prismjs/prism.js",
```

Comment:
Would this include all this stuff for the codelab page?
We may need a separate app